CONTRIBUTING.md

```diff
@@ -200,6 +200,8 @@ If the CI job fails, then you most likely need to run the commands that failed l
 This runs the unit tests in a `jsdom` environment.
 If it fails then `pnpm test:unit` should<sup>[1](test/README.md#accessibility-tree-exclusion)</sup> fail locally as well.
 You can narrow the scope of tests run with `pnpm test:unit --grep ComponentName`.
+If you encounter error: `'m)[jt]s?' is not recognized as an internal or external command` while running `pnpm test:unit`, it is likely because of an issue with how the test file extensions are handled in the script. Solution: Modify the `nx_test_unit` script in the root `package.json` to
+`"nx_test_unit": "cross-env NODE_ENV=test mocha 'packages/**/*.test.{js,jsx,ts,tsx}' 'packages-internal/**/*.test.{js,jsx,ts,tsx}' 'docs/**/*.test.{js,jsx,ts,tsx}'"` This change ensures that the test files are correctly recognized with the appropriate extensions and allows running tests using the `--grep` flag on Windows.
```

Comment:
I haven't look closed, but at first sight, this can't fly, we can't possibly ask Windows users to change the source to run the tests, it's horrible DX. 

Can we instead change the script source? It looks like this got broken in #44143.

Reply:
Yea I understand.  I tried to look further into it to see if I could have a command that runs the tests successfully on both windows and mac but I wasn't able to find one.

Reply:
I am a Windows user, adding double quotes works, as shown in [this PR](https://github.com/mochajs/mocha/pull/5219). For example:
```diff
--- a/package.json
+++ b/package.json
@@ -95,7 +95,7 @@
     "nx_test_karma": "cross-env NODE_ENV=test karma start test/karma.conf.js",
     "nx_test_regressions_run": "mocha --config test/regressions/.mocharc.js --delay 'test/regressions/**/*.test.js'
",
     "nx_test_regressions_pigment_css_run": "mocha --config apps/pigment-css-vite-app/.mocharc.cjs --delay 'apps/pig
ment-css-vite-app/**/*.test.js'",
-    "nx_test_unit": "cross-env NODE_ENV=test mocha 'packages/**/*.test.?(c|m)[jt]s?(x)' 'packages-internal/**/*.tes
t.?(c|m)[jt]s?(x)' 'docs/**/*.test.?(c|m)[jt]s?(x)'"
+    "nx_test_unit": "cross-env NODE_ENV=test mocha \"packages/**/*.test.?(c|m)[jt]s?(x)\" \"packages-internal/**/*.
test.?(c|m)[jt]s?(x)\" \"docs/**/*.test.?(c|m)[jt]s?(x)\""
   },
   "dependencies": {
     "@googleapis/sheets": "^9.3.1",
```
@ChristopherJamesL Can you try the change and confirm? It needs to be applied to other scripts too.